Formula/scrcpy.rb

```diff
@@ -0,0 +1,76 @@
+class Scrcpy < Formula
+  desc "Display and control your Android device"
+  homepage "https://github.com/Genymobile/scrcpy"
+  url "https://github.com/Genymobile/scrcpy/archive/v1.1.tar.gz"
+  sha256 "1b56caa4aad5add2c49ea436e9f26282b55a413003d0d73b029a1fbf48da0a1c"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "ffmpeg"
+  depends_on "sdl2"
+
+  resource "prebuilt-server" do
+    url "https://github.com/Genymobile/scrcpy/releases/download/v1.1/scrcpy-server-v1.1.jar"
+    sha256 "14826512bf38447ec94adf3b531676ce038d19e7e06757fb4e537882b17e77b3"
+  end
+
+  def install
+    r = resource("prebuilt-server")
+    r.verify_download_integrity(r.fetch)
+    cp r.fetch, buildpath/"prebuilt-server.jar"
+
+    system "meson", "build", "--buildtype", "release",
+      "--strip", "-Db_lto=true", "--prefix=#{prefix}",
```

Comment:
Most of our other meson-based formulas do:

```
mkdir "build" do
  system "meson", "--prefix=...", ".."
  system "ninja", "install"
end
```

So I'm wondering:
- why are we passing `build` here?
- if `--strip` and `-Db_lto=true` are not default, why use them?
- is `--buildtype release` necessary or is it the default?

(And in any case, the indentation for continuation lines is wrong)

Reply:
> if `--strip` and `-Db_lto=true` are not default, why use them?

They are not absolutely necessary, but why not enabling link-time optimization (even if meson does not enable it by default)?

> is `--buildtype release` necessary or is it the default?

The [default](http://mesonbuild.com/Running-Meson.html#configuring-the-source) is debug.